tests/unit/test_file_utils.py

```diff
@@ -330,10 +326,24 @@ def test_get_tool_from_docker_snap_path(
         assert file_utils.get_snap_tool_path("tool-command") == abs_tool_path.as_posix()
 
 
+def test_get_host_tool_finds_command(monkeypatch):
+    monkeypatch.setattr(shutil, "which", lambda x: "/usr/bin/foo")
+
+    assert file_utils.get_host_tool_path(command_name="foo", package_name="foo")
+
+
+def test_get_host_tool_failure(monkeypatch):
+    monkeypatch.setattr(shutil, "which", lambda x: None)
+    with pytest.raises(errors.SnapcraftHostToolNotFoundError) as error:
```

Comment:
this is just a nit about an empty line between test setup and actual test

Reply:
done